src/translations/en-us.json

```diff
@@ -110,7 +110,7 @@
   "SettingYourMapBaseLayer.content1": "Depending on the type of Task and the geographical area you are working on, you may want to use a different base layer for the Task map. You can quickly switch between the available layers right in the Map Widget:",
   "SettingYourMapBaseLayer.content2": "You can set your default choice for your preferred map base layer in your User Settings:",
   "SettingYourMapBaseLayer.content3": "The list of layers is retrieved from the Editor Layer Index, so the options should always be the same as those available from within JOSM and iD.\n\n## Challenge Default\n\nThe creator of the Challenge you work on may set an appropriate base layer for the map, which will be selected by default when you start working on that Challenge. You can still override this in the Map Widget.\n\n## Custom base layers\n\n_Advanced_\n\nIf you have a custom TMS tile layer source you want to use, you can select 'Custom' from the list in your User Settings, and add the layer using the same URL pattern used [in QGIS](https://docs.qgis.org/3.22/en/docs/user_manual/managing_data_source/opening_data.html#using-xyz-tile-services) and [Leaflet](https://leafletjs.com/reference.html#tilelayer), for example:",
-  "SolvingMultipleTaskTogether.content1": "Many MapRoulette Challenges have Tasks that are geographically so close together that it is more efficient to solve multiple close Tasks together as if it were one Task. MapRoulette offers a workflow for this since version 3.4.2.\n\nTo use the Multiple Tasks workflow, you need to add the **Multi-Task Work** Widget to your Task screen. To do this, click _Edit Layout_ in the right top corner of your Task screen, and use the _Add Widget_ dropdown menu to add add it to your Layout.",
+  "SolvingMultipleTaskTogether.content1": "Many MapRoulette Challenges have Tasks that are geographically so close together that it is more efficient to solve multiple close Tasks together as if it were one Task. MapRoulette offers a workflow for this since Maproulette3 version 3.4.2 and MR2 Version 3.6.4.\n\nTo use the Multiple Tasks workflow, you need to add the **Multi-Task Work** Widget to your Task screen. To do this, click _Edit Layout_ in the right top corner of your Task screen, and use the _Add Widget_ dropdown menu to add add it to your Layout.",
```

Comment:
I would avoid the MR2 reference, I assume you mean the backend version? This is not really relevant to the end user who reads this documentation.

Reply:
Thanks for catching that, got it updated.

Reply:
I actually meant removing the reference to the backend version altogether :) So I actually prefer the old wording `MapRoulette offers a workflow for this since version 3.4.2`. Just more concise and tells the end user all they need to know.